core/templates/dev/head/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js

```diff
@@ -104,6 +104,28 @@ oppia.directive('adminDevModeActivitiesTab', [
           }
         };
 
+        $scope.generateDummyExplorations = function() {
+          // Generate dummy explorations with random title.
+          AdminTaskManagerService.startTask();
+          var POSSIBLE_TITLES = ['Hulk Neuroscience',
+                                 'Quantum Starks',
+                                 'Wonder Anatomy',
+                                 'Elvish, language of "Lord of the Rings',
+                                 'The Science of Superheroes'
+          ];
+          $scope.setStatusMessage('Processing...');
+          for (var i = 0; i < $scope.DummyExplorationCount; ++i) {
+            var randomTitle = Math.floor(
+                              Math.random() * POSSIBLE_TITLES.length
+                              );
+            $http.post('/contributehandler/create_new', {
+              title: POSSIBLE_TITLES[randomTitle]
+            });
+          }
+          $scope.setStatusMessage('Dummy explorations created.');
```

Comment:
No, this won't do. $http.post() is an async call. At the point this message is shown, there is no guarantee that the explorations have been created without error. I still suggest that you do the generation in the backend.